### ISSUE TITLE
Nyromodal Problem With IE

### ISSUE THREAD

**Issue by aakalan** [nyroDev/nyroModal#161]
I'm using the Nyromodal plugin as a lightbox for a photo portfolio page on my Web site.
All works well in Firefox, Chrome and Safari, all versions.

But it is doing something strange in IE: it puts the popup modal windows below the other content on the page instead of centered and over the other content (thumbnails), as it does in the other browsers.

I tried moving my thumbnails from a table (sorry about that!) into a DIV, but with a DIV, the Nyromodal modal window shows up -under- the content of the DIV, in all browsers, instead of on top of it.

Simply put:
Without a DIV on the page, the popup shows up at the bottom of the page (as though wrapped) in IE, but correctly centered over the other content in all other browsers.

With a DIV behind my thumbnails, the popup shows up -behind- the thumbnails in all browsers.

Please help. I'm very close to abandoning this.

Thank you so much.

**Comment by aakalan** [comment]
PS: You can see the site, in action, at http://www.aakatz.com
Try it in IE and then in any other browser to see the problem.
Thank you.
Alan Katz.

**Comment by aakalan** [comment]
Got the solution!! I had an older "<DOCTYPE" tag in my headers. As soon as I changed it to the one recommended by NyroModal, all the problems went away. Sorry to waste everyone's time. I'm so relieved.... 

Alan
